File: BackEnd/postman/Paciente.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Paciente'
data={
  'accion':'crear',
  'idTipoSangre': None,
  'idPais': None,
  'noIdentidad': None,
  'pNombre': None,
  'idAscendencia': None,
  'sNombre': None,
  'sApellido': None,
  'idEstadoCivil': None,
  'direccion': None,
  'correo': None,
  'idOcupacion': None,
  'idEscolaridad': None,
  'pApellido': None,
  'sexo': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'agregarNatalidad',
  'idMadre': None,
  'ordenPartoMultiple': None,
  'idPadre': None,
  'idExpediente': None,
  'idCentroMedico': None,
  'fechaHora': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'buscarPorApellido',
  'sApellido': None,
  'pApellido': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'obtenerNumeroExpediente',
  'idPersona': None,
  'idPaciente': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'agregarDefuncion',
  'observacionCausa': None,
  'idExpediente': None,
  'fechaHora': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'buscarPorNoIdentidad',
  'noIdentidad': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'listarTodos',
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'actualizar',
  'direccion': None,
  'idOcupacion': None,
  'idPaciente': None,
  'idEstadoCivil': None,
  'correo': None,
  'idEscolaridad': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'buscarPorNombre',
  'pNombre': None,
  'sNombre': None,
}
print postear(data,url)
print

url='Paciente'
data={
  'accion':'listar',
  'idPaciente': None,
}
print postear(data,url)
print

File: BackEnd/ajax/Paciente.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idTipoSangre': null,
    'idPais': null,
    'noIdentidad': null,
    'pNombre': null,
    'idAscendencia': null,
    'sNombre': null,
    'sApellido': null,
    'idEstadoCivil': null,
    'direccion': null,
    'correo': null,
    'idOcupacion': null,
    'idEscolaridad': null,
    'pApellido': null,
    'sexo': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarNatalidad',
    'idMadre': null,
    'ordenPartoMultiple': null,
    'idPadre': null,
    'idExpediente': null,
    'idCentroMedico': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorApellido',
    'sApellido': null,
    'pApellido': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarDefuncion',
    'observacionCausa': null,
    'idExpediente': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorNoIdentidad',
    'idPersona': null,
    'idPaciente': null,
    'noIdentidad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'direccion': null,
    'idOcupacion': null,
    'idPaciente': null,
    'idEstadoCivil': null,
    'correo': null,
    'idEscolaridad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorNombre',
    'pNombre': null,
    'sNombre': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paciente.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listar',
    'idPaciente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


